babbage/query/cuts.py

```diff
@@ -27,5 +27,5 @@ def apply(self, q, cuts):
             info.append({'ref': ref, 'operator': operator, 'value': value})
             table, column = self.cube.model[ref].bind(self.cube)
             q = self.ensure_table(q, table)
-            q = q.where(column == cast(value, column.type))
```

Comment:
Nope - the value can be as incorrect as in upstream master - this only happens if the user supplies a cut value that is parsed as one type to compare to a column of another type. https://github.com/Code4SA/babbage/pull/7 will change this error message and exception to a query exception and a 400 response in the HTTP API

I got 99 problems and drunkenness ain't one. I'll supply pull requests for the rest :P